observability/src/test/java/org/acme/observability/ObservabilityTest.java

```diff
@@ -52,7 +52,7 @@ public void metrics() {
         long camelMetricCount = path.getMap("$.")
                 .keySet()
                 .stream()
-                .filter(key -> key.toString().startsWith("Camel"))
+                .filter(key -> key.toString().startsWith("camel"))
```

Comment:
makes sense, thanks for the tip (I thought that c-q-examples should be updated only with `camel-main` branch in mind, and will be merged to `main` with the new c-q....

Reply:
Still a valid point and I'll changeit

Reply:
@jamesnetherton Do we still want to marge it to camel-quarkus-main`, or should I rebase iPR to `main` -> as the change works for both?

Reply:
`camel-quarkus-main` should be enough because both the CQ `main` & `camel-main` branch CI builds will check it out.

https://github.com/apache/camel-quarkus/blob/main/.github/workflows/ci-build.yaml#L466